app/boards/shields/ergodash/ergodash-layouts.dtsi

```diff
@@ -0,0 +1,419 @@
+            , <&key_physical_attrs 100 100  600  275       0     0     0>
+            , <&key_physical_attrs 100 100 1150  275       0     0     0>
+            , <&key_physical_attrs 100 100 1250  225       0     0     0>
+            , <&key_physical_attrs 100 100 1350  212       0     0     0>
+            , <&key_physical_attrs 100 100 1450  200       0     0     0>
+            , <&key_physical_attrs 100 100 1550  212       0     0     0>
+            , <&key_physical_attrs 100 100 1650  238       0     0     0>
+            , <&key_physical_attrs 100 100 1750  238       0     0     0>
+            , <&key_physical_attrs 100 100    0  338       0     0     0>
+            , <&key_physical_attrs 100 100  100  338       0     0     0>
+            , <&key_physical_attrs 100 100  200  312       0     0     0>
+            , <&key_physical_attrs 100 100  300  300       0     0     0>
+            , <&key_physical_attrs 100 100  400  312       0     0     0>
+            , <&key_physical_attrs 100 100  500  325       0     0     0>
+            , <&key_physical_attrs 100 100  700  400       0     0     0>
+            , <&key_physical_attrs 100 100 1050  400       0     0     0>
+            , <&key_physical_attrs 100 100 1250  325       0     0     0>
+            , <&key_physical_attrs 100 100 1350  312       0     0     0>
+            , <&key_physical_attrs 100 100 1450  300       0     0     0>
+            , <&key_physical_attrs 100 100 1550  312       0     0     0>
+            , <&key_physical_attrs 100 100 1650  338       0     0     0>
+            , <&key_physical_attrs 100 100 1750  338       0     0     0>
+            , <&key_physical_attrs 100 100    0  438       0     0     0>
+            , <&key_physical_attrs 100 100  100  438       0     0     0>
+            , <&key_physical_attrs 100 100  200  412       0     0     0>
+            , <&key_physical_attrs 100 100  300  400       0     0     0>
+            , <&key_physical_attrs 100 100  500  500       0     0     0>
+            , <&key_physical_attrs 100 200  600  400       0     0     0>
+            , <&key_physical_attrs 100 100  700  500       0     0     0>
+            , <&key_physical_attrs 100 100 1050  500       0     0     0>
+            , <&key_physical_attrs 100 200 1150  400       0     0     0>
+            , <&key_physical_attrs 100 100 1250  500       0     0     0>
+            , <&key_physical_attrs 100 100 1450  400       0     0     0>
+            , <&key_physical_attrs 100 100 1550  412       0     0     0>
+            , <&key_physical_attrs 100 100 1650  438       0     0     0>
+            , <&key_physical_attrs 100 100 1750  438       0     0     0>
+            ;
+    };
+
+    ergodash_physical_layout_3key_2us: ergodash_physical_layout_3key_2us {
+        compatible = "zmk,physical-layout";
+        display-name = "LAYOUT_3key_2us";
+
+        kscan = <&kscan0>;
+        transform = <&default_transform_3key>;
```

Comment:
This seems incorrect: the key ordering shows that in this case all 6 thumb keys should be on the last row of the keymap (see image) but the matrix transform has two of the thumbs on the upper row (consistent with `LAYOUT_3key_1us` but not this one). I guess this one needs a different matrix transform, or reordered keys.

<img width="807" height="303" alt="image" src="https://github.com/user-attachments/assets/36490454-62c5-4883-a7e5-0a514b8acc9d" />

Reply:
This is good. I added electrical map, so you can see why it's like this.

Reply:
Eh.. stubs didnt worked, so i added alt keymap